filebeat/module/postgresql/log/test/postgresql-query-steps-slowlog.log-expected.json

```diff
@@ -0,0 +1,44 @@
+[
+    {
+        "@timestamp": "2019-09-04T13:52:38.004Z",
+        "event.dataset": "postgresql.log",
+        "event.duration": 12437000,
+        "event.module": "postgresql",
+        "event.timezone": "CEST",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.level": "LOG",
+        "log.offset": 0,
+        "message": "2019-09-04 15:52:38.004 CEST [31136] user@host LOG:  duration: 12.437 ms  parse <unnamed>: select * from table",
+        "postgresql.log.database": "host",
+        "postgresql.log.query": "select * from table",
```

Comment:
Should this be documented? If `postgresql.log.query_name` is <unnamed>, then it won't show up in the json output.

Reply:
Sorry, some reason the `<unnamed>` part didnt show up. Like under the documented `- name: query_name` description, do we need to add something like `Name given to a query when using extended query protocol. If it is <unnamed>, then this field will be ignored`.

Reply:
Oh, you mean to document the behaviour when no name is present, ok, I'll do it!

Reply:
Oh, I had extended the description but I prefer yours, changing it.